src/opera/template/node.py

```diff
@@ -103,5 +103,19 @@ def get_property(self, params):
             ))
         return requirements[0].target.get_property(["SELF"] + rest)
 
+    def get_attribute(self, params):
+        host, prop, *rest = params
+
+        if host != "SELF":
+            raise DataError(
+                "Accessing non-local stuff is bad. Fix your service template."
+            )
+        if host == "HOST":
+            raise DataError("HOST is not yet supported in opera.")
```

Comment:
Is this just a placeholder for the dev to remember about the HOST? Because from how I see it, this condition is never evaluated as `True`.

Reply:
That is the exact purpose of this code: to be able to grep for `HOST` and get back all of the relevant places that might need updating when/if we add support.